cardano-explorer/src/Explorer/Web/LegacyApi.hs

```diff
@@ -16,6 +16,7 @@ module Explorer.Web.LegacyApi
        ) where
 
 import           Data.Word                (Word16, Word64)
+import           Data.Int (Int64)
```

Comment:
I am pretty sure we are never going to run this on a platform where `Int` is not 64 bits, so I would thing `Int` is just fine for this if you needed a signed value.

Reply:
that was set by the type of:
```
limit :: Int64 -> SqlQuery ()
offset :: Int64 -> SqlQuery ()
```
i didnt see a need to change the type and then add `fromIntegral`s everywhere to restore the old type

Reply:
Ok, that makes sense then.